client/src/components/ReviewsPages.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+const ReviewsPages = (props) => {
+    let pages = [];
+    let counter = 1;
+    for (var i = 1; i <= props.reviews_number; i+=10) {
+        pages.push(<span className={counter} key={i} id="link" onClick={props.changeReviewsPage}>{counter}</span>);
```

Comment:
If you're storing the counter information on the html element, putting on a `data-` attribute might be more appropriate than the `className`. The `className` attribute is meant for styling, so someone might get confused about why you're using it for logic reasons. Here's some MDN articles on the `data-` attribute:
https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/data-*
https://developer.mozilla.org/en-US/docs/Learn/HTML/Howto/Use_data_attributes